bobtemplates/plone/content_type.py

```diff
@@ -143,6 +151,46 @@ def _update_rolemap_xml(configurator):
     update_file(configurator, file_path, match_str, insert_str)
 
 
+def _update_ct_types_fti_xml(configurator):
+    if (
+        (not configurator.variables.get('dexterity_type_global_allow', False)) and   # NOQA: E501
+        configurator.variables.get('dexterity_parent_container_type_name', False)    # NOQA: E501
+    ):
+        file_name = u'{0}.xml'.format(
+            configurator.variables['dexterity_parent_container_type_name'],
+        )
+        file_path = '{0}/profiles/default/types/{1}'.format(
+            configurator.variables['package_folder'],
+            file_name,
+        )
+
+        with open(file_path, 'r') as xml_file:
+            parser = etree.XMLParser(remove_blank_text=True)
+            tree = etree.parse(xml_file, parser)
+            tree_root = tree.getroot()
+            permname = """<element value="{0}" />""".format(
+                configurator.variables['dexterity_type_name'],
+            )
+            xpath_selector = ".//permission[@name='{0}']".format(permname)
```

Comment:
why are you searching for a permission tag here?

Reply:
please write a package-test, which tries to update a file, which has already the correct element in the list.
This will proably fail by now, so you can fix it then.